pkg/bindings/images/types.go

```diff
@@ -0,0 +1,9 @@
+package images
+
+//go:generate go run ../generator/generator.go ImageRemove
+// ImageRemove are optional options for image removal
+type ImageRemove struct {
```

Comment:
```suggestion
type Remove struct {
```
We know from package this is for Images.